cmd/generate/generate.go

```diff
@@ -354,12 +352,7 @@ func main() {
 			log.Println("File ", projectFilePath, " not found")
 			metadata = nil
 		}
-		// Builder image might be provided without formatting '%s' string as plain value
-		builderImage := mustGatherBuilderImageFmt
-		if strings.Count(mustGatherBuilderImageFmt, "%s") == 1 {
-			builderImage = fmt.Sprintf(mustGatherBaseImageFmt, metadata.Requirements.OcpVersion.Max)
-		}
-
+		builderImage := fmt.Sprintf(mustGatherBaseImageFmt, metadata.Requirements.OcpVersion.Max)
```

Comment:
```suggestion
		builderImage := fmt.Sprintf(mustGatherBaseImageFmt, metadata.Requirements.OcpVersion.Min)
```
I guess @mgencur s suggestion in https://github.com/openshift-knative/hack/pull/406#discussion_r1830549938 was to use `Min`.

Reply:
here i guess we used either latest or Max bcz the default quay.io seems like not so updated with oldest tag, for 4.13 build 8 months ago,
https://quay.io/repository/openshift/origin-must-gather?tab=tags

Reply:
here i guess we used either latest or Max bcz the default quay.io seems like not so updated with oldest tag, for 4.13 build 8 months ago,
https://quay.io/repository/openshift/origin-must-gather?tab=tags

Reply:
martin comment is for `brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather:v4.13` we can change it to Min instead of latest here https://github.com/openshift-knative/hack/pull/406/files#diff-8c104313544b2c9a8413d2cbb7a399e56793ffab4a3e554d5ac2243a5a74f1c8R18

Reply:
I think we can align those, so both tags are for the same OCP version.